wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListContentController/ConversationListContentController.swift

```diff
@@ -150,6 +169,26 @@ final class ConversationListContentController: UICollectionViewController {
         clearsSelectionOnViewWillAppear = false
     }
 
+    private func setupEmptyPlaceholder() {
```

Comment:
Does this solution replace the `ConversationListOnboardingHint`? Then we could clean it up as well.

Reply:
Removed